authorization/api-secret-key/README.md

```diff
@@ -0,0 +1,62 @@
+# Tapsi Pack API: API Secret Key Guide
+
+Welcome to the API Secret Key guide for the Tapsi Pack API. This guide will help you understand how to authenticate with the
```

Comment:
I'm not sure about the authenticate word in this sentence, since there is no authentication, it's just about authorization.

Reply:
fixed